### PR TITLE
[ES] Add `Notelock`

### PR DESCRIPTION
**Pull request by Ekkusu**
## Self-check

- [ ] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - jittering notes
+  - shaking notes
+  - notas bloqueadas
+  - notas temblorosas
+  - agitando notas
+---
+
+# Notelock
+
+![](img/notelock.gif "An example of notelock: the second note is inaccessible")
```
Comment (TPGPL) [inline wiki/Gameplay/Judgement/Notelock/es.md:13]: alt-texts and tooltips of images need to be translated

for reference:

`alt-text`

File: wiki/Gameplay/Judgement/Notelock/es.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+El **Notelock**, o **nota de bloqueo**, es un término informal para la mecánica del juego [osu!](/wiki/Game_mode/osu!), el cual puede evitar que un jugador elimine un objeto golpeado. Esto pasa si **dos** condiciones se cumplen al mismo tiempo:
+
+1. La [ventana de tiempo](/wiki/Beatmap``ping/Overall_difficulty#timing) de dos objetos de golpeo se superpone.
```
Comment (TPGPL) [inline wiki/Gameplay/Judgement/Notelock/es.md:17]:
```suggestion
1. La [ventana de tiempo](/wiki/Beatmapping/Overall_difficulty#timing) de dos objetos de golpeo se superpone.
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Judgement/Notelock/es.md (hunk 1 above)
Comment (Ekkusu) [reply wiki/Gameplay/Judgement/Notelock/es.md]: ok, its solved

**Comment by Ekkusu** [comment]
delete this, i make a mistake and i will make this code again (cant uptade this and i dont not know)

**Pull request closed without merging**